packages/date-time/src/components/CalendarDayGrid/CalendarDayGrid.types.ts

```diff
@@ -8,12 +8,13 @@ import {
   AnimationDirection,
 } from '../Calendar/Calendar.types';
 import { IStyle, ITheme, IProcessedStyleSet } from '@uifabric/styling';
+import { IDayGridOptions } from '@fluentui/date-time-utilities/lib/dateGrid/DateGrid.types';
 
 export interface ICalendarDayGrid {
   focus(): void;
 }
 
-export interface ICalendarDayGridProps extends IBaseProps<ICalendarDayGrid> {
+export interface ICalendarDayGridProps extends IDayGridOptions, IBaseProps<ICalendarDayGrid> {
```

Comment:
I've kept all the fields present here for the comments with defaults, since I'm not providing any defaults in `date-time-utilities`, would that be OK?

Reply:
sure i think that makes sense.